broadcast/index.md

```diff
@@ -24,12 +24,20 @@ interesting developments in the ecosystem around Trino.
 <dt>22 May 2024: Trino Community Broadcast 60 - AI functions</dt>
 <dd><a href="https://www.linkedin.com/in/isainalcik/">Isa Inalcik</a>
 chats with us a bit about his work with Trino at
-<a href="https://www.linkedin.com/company/bestsecret/">BestSecret</a>
-before we dive into his exploration of creating Trino functions that
-use generative AI systems.</dd>
-
-<dt>20 June 2024: Trino Community Broadcast 61</dt>
-<dd>Topic to be determined.</dd>
+<a href="https://bestsecret.com/">BestSecret</a> before we dive into
```

Comment:
"BestSecret before"
is kind of weird, yeah?

Reply:
you are missing the start of the sentence that is still there. This just fixes the URL